.github/workflows/monorepo-split.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
```

Comment:
I tried with v3 and it didn't work sadly, had to roll all the way back to v1 to get the correct commits 🤔 
https://github.com/SocialiteProviders/Providers/commit/e95fb7091ed1aa2ef526b4e70d157673f192b3c5

Reply:
👍